src/kokkos_ext/ArborX_DetailsKokkosExtMathFunctions.hpp

```diff
@@ -17,10 +17,20 @@
 namespace KokkosExt
 {
 
-#if KOKKOS_VERSION >= 30699
+#if KOKKOS_VERSION >= 30700
```

Comment:
Why did you change the guards?

Reply:
I actually like this change. It will work with our existing testing, and I think it's better in the medium term.

Reply:
> It will work with our existing testing

How does `30699` not work?

>  I think it's better in the medium term.

What is better about it?

Reply:
> How does `30699` not work?

That's not what I meant. I mean that 30700 would work, too.

> What is better about it?

I like hiding functionality behind the exact releases, rather than some development versions.

Reply:
Any commit after creating the release branch for 3.6 (and thus bumping the version on `develop`) would report `30699` but that doesn't mean that the changes necessary are already part of that commit.

Reply:
The promotion to the main `Kokkos::` namespace is the 1st thing that happened in the 3.7b release cycle so it is safe to do so

Reply:
That requires special knowledge about the order of commits in the release branch. :slightly_smiling_face: